resources/components/navigation.less

```diff
@@ -279,19 +279,21 @@
 #mw-sidebar-sitename {
 	position: fixed;
 	visibility: visible;
-	top: @sidebar-sitename-height + @header-height + @margin-side;
+	top: 0;
 	left: @margin-side;
 	font-size: 11px;
 	letter-spacing: 4px;
-	transform: translateY( 0 ) rotate( -90deg );
-	transform-origin: top left;
+	transform: translateY( 100% ) translateX( -100% ) rotate( -90deg );
+	transform-origin: top right;
 	transition: @transition-transform, @transition-opacity;
+	padding-right: 90px;
```

Comment:
There is no need to use padding-right with the other config i mentioned in the review